nixos/doc/manual/development/releases.xml

```diff
@@ -34,113 +36,127 @@
   <section xml:id="at-beta-release-time">
    <title>At beta release time</title>
 
-   <itemizedlist spacing="compact">
+   <orderedlist>
     <listitem>
      <para>
-      <link xlink:href="https://github.com/NixOS/nixpkgs/issues/13559">Create
-      an issue for tracking Zero Hydra Failures progress. ZHF is an effort to
-      get build failures down to zero.</link>
+      From the master branch run:
      </para>
     </listitem>
+   </orderedlist>
+
+<programlisting>
+# git checkout -b release-19.09
```

Comment:
As suggested on IRC, the `#` should be changed to `$` or dropped entirely -- `#` is root-y, which is unnecessary.